app/templates/auth/register.html

```diff
@@ -1,5 +1,5 @@
 {% extends "base.html" %}
```

Comment:
Same as login.html template....seems to be an awful lot of additional whitespace on the indentation.

Reply:
Same as above.

Reply:
Also there was a closing tag missing in both templates for a few divs.